WooCommerce/Classes/ViewRelated/Editor/AztecEditorViewController.swift

```diff
@@ -218,11 +218,35 @@ private extension AztecEditorViewController {
 
 // MARK: - Navigation actions
 //
-private extension AztecEditorViewController {
-    @objc func saveButtonTapped() {
+extension AztecEditorViewController {
+    @objc private func saveButtonTapped() {
         let content = getHTML()
         onContentSave?(content)
     }
+
+    override func shouldPopOnBackButton() -> Bool {
+        guard viewProperties.showSaveChangesActionSheet == true else {
+            return true
+        }
+
+        let editedContent = getHTML()
+        if content != editedContent {
```

Comment:
as mentioned in the CR review message, `getHTML()` from the default Rich Text View removes the HTML tags so it's always different from the original HTML. maybe we can make `content` a variable and update `content` to `getHTML()` after the editor view is set `setHTML(content)`?